dotcom-rendering/src/components/ListenToArticle.importable.tsx

```diff
@@ -30,23 +33,45 @@ export const ListenToArticle = ({ articleId }: Props) => {
 		number | undefined
 	>(undefined);
 
-	const isBridgetCompatible = useIsBridgetCompatible('8.6.0');
+	const isBridgetCompatible = useIsBridgetCompatible('8.7.0');
 	useEffect(() => {
 		if (isBridgetCompatible) {
 			Promise.all([
+				// AB TESTING native
+				getNativeABTestingClient().getParticipations(),
 				getListenToArticleClient().isAvailable(articleId),
 				getListenToArticleClient().isPlaying(articleId),
 				getListenToArticleClient().getAudioDurationSeconds(articleId),
 			])
-				.then(([isAvailable, isPlaying, durationSeconds]) => {
-					setAudioDurationSeconds(
-						typeof durationSeconds === 'number' &&
-							durationSeconds > 0
-							? durationSeconds
-							: undefined,
-					);
-					setShowButton(isAvailable && !isPlaying);
-				})
+				.then(
+					([
+						AbParticipations,
+						isAvailable,
+						isPlaying,
+						durationSeconds,
+					]) => {
+						// AB TESTING native start
+						const variant = AbParticipations.get(
```

Comment:
Small thing, but PascalCase is usually reserved for types (including classes), whereas this is an instance of a `Map`, so can probably be camelCase to avoid confusion:

```suggestion
						abParticipations,
						isAvailable,
						isPlaying,
						durationSeconds,
					]) => {
						// AB TESTING native start
						const variant = abParticipations.get(
```